### PR TITLE
[APRComp24] RepairCat-Functional-Errors

### PR DESCRIPTION
**Pull request by jiang719**

### CODE REVIEW

File: app/core/values.py
Comment (Marti2203) [inline app/core/values.py:1]: Hi, This is changed from our side, please do remove this modification from the PR

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# [APRComp24] RepairCat - Functional Errors
```
Comment (Marti2203) [inline README.md:1]: Can this also be removed :sweat_smile:

Comment (jiang719) [reply README.md]: sure, removed

Comment (Marti2203) [reply README.md]: Thanks!

**Comment by rshariffdeen** [comment]
@jiang719 the PR is not complete, can you update the digest of the image in the driver?

**Comment by jiang719** [comment]
> @jiang719 the PR is not complete, can you update the digest of the image in the driver?

added

**Comment by Marti2203** [comment]
Thanks! Can you also change the branch to merge to be dev?

**Comment by jiang719** [comment]
> Thanks! Can you also change the branch to merge to be dev?

We have two branches, for the two tracks. Do you mean combine them together to one branch "dev"?

**Comment by Marti2203** [comment]
Sorry for the ambiguity. I meant that currently, your PR is to the master branch of the project. You can change the merge to the dev branch of this project through the edit button at the top.

**Comment by Marti2203** [comment]
Thanks!

**Pull request merged**